src/test/java/utilities/Util.java

```diff
@@ -12,5 +16,19 @@ public static void wait(int seconds) {
 			e.printStackTrace();
 		}
 	}
+	
+	/*
+	 * sendKey function
+	 * Author:Joyce
+	 */
+	public static void sendKey(WebDriver driver,String value,String xpathLocator) {
+		WebElement ele = driver.findElement(By.xpath(xpathLocator));
```

Comment:
Instead of passing in xpathLocator, pass in a `By` object would make it more reusable